src/rgw/rgw_asio_client.h

```diff
@@ -4,61 +4,25 @@
 #define RGW_ASIO_CLIENT_H
 
 #include <boost/asio/ip/tcp.hpp>
-#include <beast/http/message.hpp>
-#include <beast/http/message_parser.hpp>
-#include <beast/core/flat_streambuf.hpp>
+#include <boost/beast.hpp>
```

Comment:
You're bringing in websocket and zlib headers too, this could slow down your build by a tiny bit. You could instead write
```
#include <boost/beast/http.hpp>
```
Up to you